express/properties/workflow.py

```diff
@@ -1,6 +1,178 @@
+                    unit["value"] = "True"
+
+                # Remove workflow property, so predict runs don't return another workflow
+                elif {"name": "workflow:pyml_predict"} in unit["results"]:
+                    unit["results"].remove({"name": "workflow:pyml_predict"})
+
+
+        return predict_subworkflows
+
+    def _construct_predict_subworkflow_units(self, train_subworkflow_units: list) -> list:
+        """
+        Constructs the predict subworkflow units, for use in the generated predict workflow.
+        Here, and only here, "units" is intended to mean "subworkflows," because that's what the Workflows object
+        uses to define the subworkflow graph.
+
+        Args:
+            train_subworkflow_units: "Units" defined in the original workflow.
+
+        Returns:
+            A list describing the order in which subworkflows will be executed.
+        """
+        # Todo: Implement the way we'll construct the subworkflow units below
+        return train_subworkflow_units
+
+    def _serialize(self):
+        """
+        Creates the actual ML Predict workflow that will be output from a job. Intended for the quick implementation.
+        """
+        # Construct the "units" key inside the workflow. Here (and only here), "units" actually means "subworkflows,"
+        # because that's what the key is called inside "workflow"
+        train_subworkflow_units: list = self.workflow["units"]
+        predict_subworkflow_units = self._construct_predict_subworkflow_units(train_subworkflow_units)
+
+        # Construct the "subworkflows" key inside the workflow
+        train_subworkflows: list = self.workflow["subworkflows"]
+        predict_subworkflows = self._construct_predict_subworkflows(train_subworkflows)
+
+        # Create the workflow
+        workflow = {
+            # Contents of the workflow:
+            "units": predict_subworkflow_units,
+            "subworkflows": predict_subworkflows,  # Units which make up each subworkflow are found here
+
+            # Metadata describing the workflow
+            "name": self.name,  # Friendly name for the workflow
+            "creator": {  # Information about the account that created this workflow ("" for automatic filling)
+                "_id": "",
+                "cls": "User",
+                "slug": ""
+            },
+            "owner": {  # Information about the account that currently owns this workflow ("" for automatic filling)
+                "_id": "",
+                "cls": "Account",
+                "slug": ""
+            },
+            "schemaVersion": "0.2.0",  # Version of this schema to use in ESSE
+            "exabyteId": "",  # ID for the corresponding bank workflow. Leave as the empty string.
+            "hash": "",  # Hash used to compare workflows for uniqueness. Leave as the empty string.
+            "_id": "",  # ID for MongoDB; needs to be blank. Leave as the empty string.
+        }
```

Comment:
Move lines 156 to 169 to the abstract class and reuse here and in the ExabyteMLPredictWorkflow below

Reply:
These have been moved to the parent class